### DISCUSSION TITLE
Existing UI library build on top of the Restyle

### DISCUSSION THREAD

**Discussion by lasharela** [Shopify/restyle#194] [Q&A]
Is there any open source UI library which is build on top of the Restyle?
Also, have feeling Restyle is no longer supported by Shopify. is this true?

I like the concept. It's very sad, there is no more activity and commendation

**Comment by fortmarek** [comment]
> Is there any open source UI library which is build on top of the Restyle?

I don't know of any but would be curious to know if there was one! You can check out the current open source users of the library [here](https://github.com/Shopify/restyle/network/dependents).

> Also, have feeling Restyle is no longer supported by Shopify. is this true?

No, Restyle is still heavily used at Shopify and something we will continue on supporting (now with more focus also on external requests)
